src/components/Search/envVarSetup.js

```diff
@@ -67,12 +29,102 @@ const constructAssetsParam = (collection, tilerParams) => {
   return [assets[0], `assets=${assets.join('&assets=')}`]
 }
 
+// construct assets params from env variables for mosaic mode
 export const constructMosaicAssetVal = (collection) => {
-  const asset = getTilerParams()[collection]?.mosaic_asset || ''
+  const asset = getTilerParams(envMosaicTilerParams)[collection]?.assets || ''
   if (!asset) {
     console.log(`Assets not defined for ${collection}`)
     return null
   } else {
-    return asset
+    return asset.toString()
   }
 }
+
+// method to construct tiler parameter values for scene and mosaic
+const parameters = {
+  colorFormula: function (tilerParams, collection) {
```

Comment:
I prefer fat-arrow functions in cases like this

```suggestion
  colorFormula: (tilerParams, collection) => {
```

Reply:
This has been resolved.